### DISCUSSION TITLE
Busy indicator alters the width of analytical table

### DISCUSSION THREAD

**Discussion by Wint81** [UI5/webcomponents#2718] [Q&A] [answered]
Hello,

When surrounding the analytical table with a busy indicator the width of the table is altered.
Please find example at [code sandebox exampe](https://codesandbox.io/s/exciting-water-eeuth?file=/src/App.js)
Am I missing a style property or else for the busy indicator?

Thanks and best regards

**Comment by ilhan007** [comment]
Hello @Wint81 

All the 3 tables in the sample look the same to me, could you be more descriptive what is the issue, snapshot will help to point it out.
But in general, when wrapping block elements, we usually set `display: block` to the ui5-busyindicator.

**Comment by Wint81** [comment]
Hi @ilhan007 ,

Sorry I forgot to mention that this especially happens on resize as per screenshot below:
![image](https://user-images.githubusercontent.com/77397740/105412023-72dbe300-5c34-11eb-82e1-3959c293fdd0.png)
Here I just made the rendering/preview panel larger once the page was loaded. As you can see with screenshot above, tables end up in different sizes.

Best regards

**Comment by ilhan007** [comment]
Set `width: 100%` to your slot - the root "div" element of your table. BTW, You can consider using ui5-table.

**Comment by Wint81** [comment, marked as answer]
Hey @ilhan007 ,
Seems to get fixed when I set the width to the actual table but not the parent container.
Thanks for your support.
